### PR TITLE
Rename repository from SickRage > Medusa

### PR DESCRIPTION
**Pull request by fernandog**
## BEFORE MERGING THIS WE NEED TO RENAME REPOSITORY
otherwise it will break the  `readme.md`

After merging this:
- [x] Rename repository in Github.
- [x] Rename sickrage.io repo
- [x] Change all Sickrage referances in the github.io repo to Medusa

TODO:
- [ ] Replace git_remote_url when restoring a config.ini
- [x] Respect user protocol URL from git_remote_url
- [x] Remove legacy setting `trakt_remove_show_from_sickrage`
- [x] Rename sickrage.log files @ratoaq2 
- [x] Rename database file
- [x] Change backup/restore naming pattern to medusa

https://help.github.com/articles/renaming-a-repository/
_When you rename a repository, all existing information, with the exception of Project Pages URLs, is _**automatically redirected to the new name**_

_In addition to redirecting web traffic, all **git clone, git fetch, or git push operations targeting the previous location will continue to function as if made on the new location**. However, to reduce confusion, we strongly recommend updating any existing local clones to point to the new repository URL_

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
We should also add a script to rewrite people's git config from `pymedusa/sickrage` to `pymeudsa/medusa`.

File: medusa/__init__.py [hunk 1]
```diff
@@ -70,25 +70,29 @@
...
 DONATIONS_URL = '{0}/wiki/Donations'.format(APPLICATION_URL)
 WIKI_URL = '{0}/wiki'.format(APPLICATION_URL)
 GITHUB_IO_URL = 'http://github.com/PyMedusa/sickrage.github.io/'
 EXTRA_SCRIPTS_URL = '{0}/wiki/Post-Processing#extra-scripts'.format(APPLICATION_URL)
 SUBTITLES_URL = '{0}/wiki/Subtitle%20Scripts'.format(APPLICATION_URL)
+# TODO: create an app in pushover
 PUSHOVER_URL = 'https://pushover.net/apps/clone/sickrage'
-RARBG_APPID = 'sickrage2'
-SECURE_TOKEN = 'sickrage_user'
+RARBG_APPID = 'medusa'
```
Comment (fernandog) [inline medusa/__init__.py]: @rarbg do we need to register that or just use it?

File: medusa/__init__.py [hunk 2]
```diff
@@ -748,8 +752,8 @@ def initialize(consoleLogging=True):  # pylint: disable=too-many-locals, too-man
         GIT_REMOTE_URL = check_setting_str(CFG, 'General', 'git_remote_url',
                                            'https://github.com/%s/%s.git' % (GIT_ORG, GIT_REPO))
 
-        if 'com/sickrage' in GIT_REMOTE_URL.lower():
-            GIT_REMOTE_URL = 'https://github.com/PyMedusa/SickRage.git'
+        if 'com/sickrage' in GIT_REMOTE_URL.lower() or 'sickrage.git' in GIT_REMOTE_URL.lower():
+            GIT_REMOTE_URL = 'https://github.com/pymedusa/Medusa.git'
```
Comment (fernandog) [inline medusa/__init__.py]: this is where we change current git_remote_url
the first part of the if was done in SICKRAGETV migration

File: medusa/__init__.py [hunk 3]
```diff
@@ -157,7 +161,7 @@
 GIT_PATH = None
 DEVELOPER = False
 
-NEWS_URL = 'https://cdn.pymedusa.com/sickrage-news/news.md'
+NEWS_URL = 'https://cdn.pymedusa.com/medusa-news/news.md'
```
Comment (fernandog) [inline medusa/__init__.py]: @OmgImAlexis we will need to rename github-io repo too

File: medusa/__init__.py (hunk 1 above)
Comment (rarbg) [inline medusa/__init__.py]: @fernandog just use it :)

**Comment by ratoaq2** [comment]
# WARNING
**_When running this branch, your database, backups and logs will be renamed (not lost, just renamed)
Your git url will be updated as well and the new repository url is still not ready_**

**Comment by fernandog** [comment]
Tested:
dbs OK
logs OK
had to login again (cookie sickrage-user > medusa-user) OK
NEWs OK,
changelog OK,
git_remote_url OK

waiting approval or if someone wants to test before approve

**Comment by ripper1967** [comment]
Everything goes ok except i had to redo all my config  it keeps renaming it  when i transfer it

**Comment by fernandog** [comment]
@ripper1967 what do you mean by that?

**Comment by ripper1967** [comment]
i can install and transfer  everything over  the db and images cant transfer the config file  if i do it  renames my config i transfered  and defaults  a new config

**Comment by fernandog** [comment]
weird. where is the path of your config.ini ?
Mine is inside /Medusa/ folder together with db and stuff

**Comment by ripper1967** [comment]
when restore, could it change the git_remote_url cause it points to the original  unless you manually change it

File: setup.py [hunk 4]
```diff
@@ -10,12 +10,11 @@
...
 
 setup(
-    name="sickrage",
-    version="0.0.1",
+    name="medusa",
     description="Automatic Video Library Manager for TV Shows",
     long_description=long_description,
     packages=find_packages(),
-    install_requires=['six', 'requests', 'tornado', 'profilehooks', 'mako', 'pygithub', 'contextlib2', ],
+    install_requires=['six', 'profilehooks', 'contextlib2', ],
```
Comment (fernandog) [inline setup.py]: @ratoaq2 @medariox @p0psicles any of you knows what is the use for this setup.py ?

**Comment by fernandog** [comment]
@rarbg hey, do you have some stats from `medusa` app users ?

**Pull request merged**